src/main/java/com/bobocode/creditadvisoryapp/rest/AssignmentController.java

```diff
@@ -0,0 +1,25 @@
+package com.bobocode.creditadvisoryapp.rest;
+
+import com.bobocode.creditadvisoryapp.dto.AssignmentRequest;
+import com.bobocode.creditadvisoryapp.service.AssignmentService;
+import com.bobocode.creditadvisoryapp.service.AssignmentServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/assign")
+public class AssignmentController {
+
+  private final AssignmentService assignmentService;
+
+  @PostMapping
+  public ResponseEntity<?> assign(@RequestBody AssignmentRequest request) {
```

Comment:
It is not a good practice to use wildcards in a return types. In current case a type of the body is String, so you can use ResponseEntity<String>